### DISCUSSION TITLE
Data Quality Monitor at RCNP GR

### DISCUSSION THREAD

**Discussion by shinsuke-ota** [spadi-alliance/nestdaq-user-impl#5] [Announcements]
どんなデータクオリティモニターが必要か?
どうモニターするか。

すでに存在する ELPH のソースコードから、既存のものを拾い上げてみる

**Comment by sirotori1119** [comment]
ELPHのテスト実験でのモニターをおさらい。
SubTimeFrameBuildしたデータに対して、6台のHULモジュール(GRのANAMEQに相当)の以下の情報をモニター。
　・HeartBeat: 各6台モジュールでの数とモジュールごとの時系列
　・Error Recovery : 各6台モジュールでの数とモジュールごとの時系列
　・HeartBeat + Error Recovery : 各6台モジュールでの数とモジュールごとの時系列
　・Spill end: 各6台モジュールでの数
モニターにはRootのHttpServerを利用。ブラウザ上でチェックした。
